backend/llm/openai_functions.py

```diff
@@ -1,23 +1,42 @@
 from typing import Any, Dict, List, Optional
 
 from langchain.chat_models import ChatOpenAI
-from llm.brainpicking import BrainPicking
+from llm.models.FunctionCall import FunctionCall
+from llm.models.OpenAiAnswer import OpenAiAnswer
 from logger import get_logger
+from models.chat import ChatHistory
 from repository.chat.get_chat_history import get_chat_history
+from repository.chat.update_chat_history import update_chat_history
 from vectorstore.supabase import CustomSupabaseVectorStore
 
-from .utils.format_answer import format_answer
+from .base import BaseBrainPicking
 
 logger = get_logger(__name__)
 
 
-class BrainPickingOpenAIFunctions(BrainPicking):
-    DEFAULT_MODEL = "gpt-3.5-turbo-0613"
-    DEFAULT_TEMPERATURE = 0.0
-    DEFAULT_MAX_TOKENS = 256
+def format_answer(model_response: Dict[str, Any]) -> OpenAiAnswer:
```

Comment:
What do you think About creating a class BaseAnswerModel  (with `_format_answer` )and then BaseOpenAiAnswer, VertexAnswer, ClaudeAnswer, PrivateLLMAnswer etc ? (for a next PR of course)

Reply:
Could be good. I'm not sure what functionality or readability that would bring to have another sub class?